translations/catroweb.en.yml

```diff
@@ -66,6 +66,14 @@ programs:
     There is no way to restore your actions!
     Yes, delete it!
     Cancel
+
+  changeVisibility: |+
+    Are you sure?
+    If you set "%programName%" to private, it will not be seen or found by other users.
+    You can still share the link of your project with your friends. There is no way to restore your actions!
```

Comment:
Actually it can be easily restored. Just click the button again.

My recommendation would be to change it to a hint that it can be easily changed again later.